### ISSUE TITLE
Don't understand instructions for thumbnail

### ISSUE THREAD

**Issue by designbysue** [d-matt/piwigo-jplayer#13]
I am using the current version of piwigo and have folders that were ftped to the site to create albums.  If I put the mp4 files in the folder, the video plays - and that is great - but I am trying to add the thumbnail and can't get it to work.

1.  I am guessing  pwg_representative is a folder - where is that folder to be located?

2.  Is the naming "base_name.jpg " to be the same name as the video - or how do the files get associated with each other?

So basically I get installing the video but how do I associate a thumbnail with it?  The block with a question mark doesn't work for me.

Thanks for the plug in - I like it and will love it if I can get the thumbnails working

Sue

**Comment by designbysue** [comment]
Finally got this working - for others - yes you create a folder within the folder where the video is  and name it pwg_representative.  Then upload your video thumbnail to that folder, naming it with the same name as your image.    The secret to getting this working seems to be do not use any special characters, including capital letters.  Once I named each file with a simple name with all lower case letters it worked.  Hope this helps others and saves them hours of searching for the answer (as I did)

Sue
